qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -56,9 +56,51 @@ def _wait_for_results(self, timeout: float, wait: float = 5) -> List[Dict[str, D
         """
 
         self.wait_for_final_state(timeout, wait)  # should call self.status()
-
         return [self._job_info[job_id] for job_id in self._job_id.split(",")]
 
+    def _arrange_counts(
```

Comment:
we should probably have tests for these helper methods (at the very least a test for _arrange_counts)

Reply:
Added some more tests with [474d162](https://github.com/Infleqtion/client-superstaq/pull/636/commits/474d1628ed419ca79f838ee083b0926aff9f9979)